build.env

```diff
@@ -36,7 +36,7 @@ SNAPSHOT_VERSION=v3.0.1
 HELM_VERSION=v3.1.2
 
 # minikube settings
-MINIKUBE_VERSION=v1.17.1
+MINIKUBE_VERSION=v1.18.0
```

Comment:
@Madhu-1 do we need to update minikube with this change?

Reply:
@humblec am updating minikube to check is there any issue in the kubernetes network plugins  as the tests are timing out

Reply:
Oh. .Ok.. thanks for the confirmation